src/bokeh/util/serialization.py

```diff
@@ -30,6 +30,7 @@
 # Standard library imports
 import datetime as dt
 import uuid
+from datetime import timezone
```

Comment:
Again, we only want a single `datetime` import.

Reply:
Modified the imports here too